packages/markdown/components/Heading/index.jsx

```diff
@@ -1,14 +1,5 @@
 const React = require('react');
 const PropTypes = require('prop-types');
-const kebab = require('lodash/kebabCase');
```

Comment:
Do we still need lodash?

Reply:
Yup, it's being used in section hash hrefs!

Reply:
Ah, missed that you moved it.

Reply:
In one of our plugin transformers.